### PR TITLE
Adopt a more common version tracking scheme

### PR DESCRIPTION
**Pull request by bbatsov**
Using a `Version` module is pretty much the norm for Ruby gems (+ it reduces a lot of code duplication).

### CODE REVIEW

File: lib/volt/cli/new_gem.rb [hunk 1]
```diff
@@ -103,8 +103,7 @@ def gem_options
   end
 
   def volt_version_base
-    version_path = File.join(File.dirname(__FILE__), '../../../VERSION')
-    File.read(version_path).split('.').tap { |v| v[v.size - 1] = 0 }.join('.')
+    Volt::Version::STRING.split('.').tap { |v| v[v.size - 1] = 0 }.join('.')
```
Comment (bbatsov) [inline lib/volt/cli/new_gem.rb]: This can also be moved to the `Version` module.

**Comment by ryanstout** [comment]
Originally there was a reason for the way we did things, but its no longer an issue.  Would you mind updating the PR and I'll merge?

**Comment by bbatsov** [comment]
Done.

**Comment by ryanstout** [comment]
I merged this in by hand because more things had changed.  Thanks!

**Pull request closed without merging**